### PR TITLE
GitHub issue fixes and misc edits

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[a45333f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/a45333f15a71d20fba6188bd3e83b0722536d5d1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1628) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908141625400291-1628%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=m0OCMbIf%2BO0GnIbspClnEiw1oM26eIGG2kNe1AZ2DTg%3D&st=2019-08-14T16%3A21%3A37Z&se=2019-09-14T16%3A26%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
@zivkan Can you take a look at the changes in the multitargeting article?

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[372a444](https://github.com/NuGet/docs.microsoft.com-nuget/commits/372a44437399a092814154d1a8993dfe6d8ccc61)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1628) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1628) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908141847189189-1628%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Dluy8oIlOz%2F5kvKg1upIR%2BAh%2BTin6x5D6a2DnyuyAGU%3D&st=2019-08-14T18%3A43%3A13Z&se=2019-09-14T18%3A48%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[b8e96f0](https://github.com/NuGet/docs.microsoft.com-nuget/commits/b8e96f0c785e64e5f741e6aa2689c3185fd22831)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1628) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1628) |
[docs/reference/errors-and-warnings/NU5100.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/reference/errors-and-warnings/NU5100.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5100?branch=pr-en-us-1628) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908141857513005-1628%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JntdKFJF50hnNFD1F99eW1y1J0JmZwjcysnPmr6Iba0%3D&st=2019-08-14T18%3A53%3A48Z&se=2019-09-14T18%3A58%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Supporting-Multiple-Target-Frameworks.md [hunk 1]
```diff
@@ -15,6 +15,32 @@ This article describes the layout of a NuGet package, regardless of how the pack
 
 You must manually lay out the package as described in this article when using the convention-based working directory method described in [Creating a package](../create-packages/creating-a-package.md#from-a-convention-based-working-directory). For an SDK-style project, the automated method is recommended, but you may also choose to manually lay out the package as described in this article.
 
+## Declaring dependencies
```
Comment (zivkan) [inline docs/create-packages/Supporting-Multiple-Target-Frameworks.md]: We should mention that packing a project file, NuGet will try to automatically generate the dependencies from the project. The information below about using a nuspec file should only be necessary for advanced scenarios.

File: docs/create-packages/Supporting-Multiple-Target-Frameworks.md [hunk 2]
```diff
@@ -15,6 +15,32 @@ This article describes the layout of a NuGet package, regardless of how the pack
...
+
+Each group has an attribute named `targetFramework` and contains zero or more `<dependency>` elements. Those dependencies are installed together when the target framework is compatible with the project's framework profile. See [Target frameworks](../reference/target-frameworks.md) for the exact framework identifiers.
+
+When no `targetFramework` attribute is used, the `<group>` element is used as the default or fallback list of dependencies.
+
+The following example shows different variations of the `<group>` element:
+
+```xml
+<dependencies>
+    <group>
```
Comment (zivkan) [inline docs/create-packages/Supporting-Multiple-Target-Frameworks.md]: I don't know what happens when one of the groups does not have a `targetFramework`. I would remove it, and advice that we recommend having one group per tfm in the `lib/` and `ref/` folders (in fact, nuget 5.3 should have a new validation that will warn when the asset tfms and dependencies tfms do not match).

File: docs/create-packages/Supporting-Multiple-Target-Frameworks.md (hunk 1 above)
Comment (Mikejo5000) [reply docs/create-packages/Supporting-Multiple-Target-Frameworks.md]: Thx, updating

File: docs/create-packages/Supporting-Multiple-Target-Frameworks.md (hunk 2 above)
Comment (Mikejo5000) [reply docs/create-packages/Supporting-Multiple-Target-Frameworks.md]: Updating

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[65bd209](https://github.com/NuGet/docs.microsoft.com-nuget/commits/65bd20999316ccb6a98cf566df8e4be63d788e13)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1628) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1628) |
[docs/reference/errors-and-warnings/NU5100.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/reference/errors-and-warnings/NU5100.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5100?branch=pr-en-us-1628) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908142058022000-1628%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=w7f92oBdp2m3Cv2VL30Yaom6sfqxBqxu34rMNmqhYC8%3D&st=2019-08-14T20%3A54%3A07Z&se=2019-09-14T20%3A59%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**